pkg/virt-handler/vm_test.go

```diff
@@ -1916,8 +1916,11 @@ var _ = Describe("VirtualMachineInstance", func() {
 
 			interfaceName := "interface_name"
 			mac := "C0:01:BE:E7:15:G0:0D"
-			ip := "2.2.2.2/24"
-			ips := []string{"2.2.2.2/24", "3.3.3.3/16"}
+			cidr := "2.2.2.2/24"
+			cidrs := []string{"2.2.2.2/24", "fe80::3825:65ff:fe6e:1ee9/64"}
```

Comment:
for completion, can we add an IP to this list that doesn't have a mask as well? 

Your logic checks out as it is now, this request is to ensure we future proof this a bit if someone else touches the logic.

Reply:
good point, thanks for review, I will do as you suggest.